heron/config/src/yaml/conf/yarn/uploader.yaml

```diff
@@ -1,5 +1,5 @@
 # uploader class for transferring the topology jar/tar files to storage
-heron.class.uploader:                            com.twitter.heron.uploader.NullUploader
+heron.class.uploader:                            com.twitter.heron.uploader.localfs.LocalFileSystemUploader
```

Comment:
The YARN scheduler does not need to upload heron core, A use case for a `no-op` uploader. Replacing it with this uploader will result in unnecessary copies of the binaries. Will it be ok to let the `NullUploader` exist?